### ISSUE TITLE
Weird positioning of projection boxes

### ISSUE THREAD

**Issue by rlisahuang** [UCSD-PL/vscode#24]
To replicate, paste the following code into the web editor:
```py
import numpy as np
"""
String Compression:
    Implement a method for basic string
    compression, using the count of
    repeated characters. You can assume
    that the string is non-empty, and
    only contains alphabetic characters.
    >>> task('aabccca')
    '2a1b3c1a'
"""


def task(s):
    rs = ''

    return rs

a = np.array([1,2,3])
b = 20
```

You will get:
<img width="775" alt="Screenshot 2023-02-21 at 9 51 41 AM" src="https://user-images.githubusercontent.com/29112035/220393902-53ba823e-5a73-4ae4-af21-077b2c8d5701.png">

**Comment by rlisahuang** [comment]
Removing the `import` statement solves the issue. Seems like the bug has to do with imports.

**Comment by rlisahuang** [comment]
```py
import numpy as np


def task(s):
    rs = ''

    return rs

a = np.array([1,2,3])
b = 20
```
This solves the issue. So the bug really has to do with the import + docstring combo.

**Comment by rlisahuang** [comment]
Fixed in 3a1ea90c8804c812aa6404fd6fcb88610aa5d097.
